src/h2_stream.erl

```diff
@@ -114,10 +114,10 @@ start_link(StreamId, Connection, CallbackModule, Socket) ->
 send_pp(Pid, Headers) ->
     gen_fsm:send_event(Pid, {send_pp, Headers}).
 
--spec send_frame(pid(), http2_frame:frame()) ->
+-spec send_data(pid(), http2_frame:frame()) ->
```

Comment:
send_frame sounds more generic. This only works for data frames, and I changed recv_frame to recv_data a while ago.